apps/notification/views.py

```diff
@@ -71,3 +75,29 @@ def get_count(self, request, version=None):
             'total': total,
         }
         return response.Response(result)
+
+
+class AssignmentViewSet(viewsets.ModelViewSet):
+    serializer_class = AssignmentSerializer
+    permission_classes = [permissions.IsAuthenticated]
+    filter_backends = (django_filters.rest_framework.DjangoFilterBackend,)
+    filterset_class = AssignmentFilterSet
+    pagination_class = AssignmentPagination
+
+    def get_queryset(self):
+        return Assignment.get_for(self.request.user).select_related(
+            'project', 'created_by', 'content_type',
+        )
```

Comment:
@k9845 Could you check if select_related is working here?. Basically enable silk, add `USE_SILK=True` to .env and `docker-compose up -d server`. Then you will be able to see the request /silk.

Then hit the API endpoint and check if there is n+1 query.

Reply:
Checked with silk and found no any n+1 query. @thenav56

Reply:
Super